packages/signers/src/lit-protocol/types.ts

```diff
@@ -0,0 +1,35 @@
+import type { SmartAccountAuthenticator } from "@alchemy/aa-core";
+import type { LitNodeClient } from "@lit-protocol/lit-node-client";
+import {
+  type AuthMethod,
+  type SessionSigsMap,
+  type SessionKeyPair,
+} from "@lit-protocol/types";
+
+export type LitAuthMethod = AuthMethod;
+export type LitSessionSigsMap = SessionSigsMap;
+export type LitUserMetadata = LitSessionSigsMap;
+export interface LitConfig {
+  pkpPublicKey: string;
+  rpcUrl: string;
+  inner?: LitNodeClient;
+  network?: string;
+  debug?: boolean;
+}
+
+export interface LitAuthenticateProps<
+  C extends LitAuthMethod | LitSessionSigsMap
+> {
+  context: C;
+  expiration?: string;
+  sessionKeypair?: SessionKeyPair;
+  chain?: string;
```

Comment:
can we use Viem's chain object? that way the dev doesn't need to spell out the chains.

you can also do a typed string e.g. `chain?: "ethereum" | "polygon"` but the viem approach is easier.

Reply:
feel free to push back since it seems you have a helper in the lit/constants library. just want to avoid that lint error ignore you have